.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: false
+
+os:
+  - osx
+  - linux
+
+language: node_js
+node_js: lts/*
+
+before_install:
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
+      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
+      sh -e /etc/init.d/xvfb start;
+      sleep 3;
+    fi
+
+install:
+  - npm install
```

Comment:
This needs to be `npm install --no-optional`.

Reply:
The point of optional dependencies is that they are _optional_... i.e., you _don't_ need to specify that flag.